module-system/sov-modules-core/src/storage.rs

```diff
@@ -1,24 +1,26 @@
-use std::fmt::Display;
```

Comment:
The file hierarchy is too flat, and we should split it logically into several rust modules:
- `storage, witness, cache, scratchpad` 
- `key, codec`
- everything else

Reply:
Good suggestion; divided into 2 levels